### ISSUE TITLE
Config tab in Service Bodies Just Spins When Logged in as ASC

### ISSUE THREAD

**Issue by californiasteve** [bmlt-enabled/yap#263]
**Describe the bug**
When logged in as an ASC after clicking the "Config" button under Service Bodies spinner just rotates.

**What version are you running?**
Tested on both 3.00 and 3.0.1, also tested on two different servers/providers

**Did this happen on prior versions to the one you are running?**
Yes
**To Reproduce**
Login as an ASC, go to Service Bodies, click on Area Service Body

**Expected behavior**
Config parameters open up.

**Comment by radius314** [comment]
are there any javascript errors?

**Comment by californiasteve** [comment]
I'm not seeing any.

**Comment by radius314** [comment]
able to repro, misread your ticket about being logged in as an ASC admin

**Comment by radius314** [comment]
Here is the error that is causing the issue:
```
<br />
<font size='1'><table class='xdebug-error xe-notice' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Notice: Trying to get property of non-object in /var/www/html/admin/auth_verify.php on line <i>17</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th></tr>
<tr><th align='center' bgcolor='#eeeeec'>#</th><th align='left' bgcolor='#eeeeec'>Time</th><th align='left' bgcolor='#eeeeec'>Memory</th><th align='left' bgcolor='#eeeeec'>Function</th><th align='left' bgcolor='#eeeeec'>Location</th></tr>
<tr><td bgcolor='#eeeeec' align='center'>1</td><td bgcolor='#eeeeec' align='center'>0.0183</td><td bgcolor='#eeeeec' align='right'>242488</td><td bgcolor='#eeeeec'>{main}(  )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>0</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>2</td><td bgcolor='#eeeeec' align='center'>0.0412</td><td bgcolor='#eeeeec' align='right'>252064</td><td bgcolor='#eeeeec'>require_once( <font color='#00bb00'>'/var/www/html/admin/auth_verify.php'</font> )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>2</td></tr>
</table></font>
<br />
<font size='1'><table class='xdebug-error xe-notice' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Notice: Trying to get property of non-object in /var/www/html/admin/auth_verify.php on line <i>17</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th></tr>
<tr><th align='center' bgcolor='#eeeeec'>#</th><th align='left' bgcolor='#eeeeec'>Time</th><th align='left' bgcolor='#eeeeec'>Memory</th><th align='left' bgcolor='#eeeeec'>Function</th><th align='left' bgcolor='#eeeeec'>Location</th></tr>
<tr><td bgcolor='#eeeeec' align='center'>1</td><td bgcolor='#eeeeec' align='center'>0.0183</td><td bgcolor='#eeeeec' align='right'>242488</td><td bgcolor='#eeeeec'>{main}(  )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>0</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>2</td><td bgcolor='#eeeeec' align='center'>0.0412</td><td bgcolor='#eeeeec' align='right'>252064</td><td bgcolor='#eeeeec'>require_once( <font color='#00bb00'>'/var/www/html/admin/auth_verify.php'</font> )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>2</td></tr>
</table></font>
<br />
<font size='1'><table class='xdebug-error xe-notice' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Notice: Trying to get property of non-object in /var/www/html/admin/auth_verify.php on line <i>17</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th></tr>
<tr><th align='center' bgcolor='#eeeeec'>#</th><th align='left' bgcolor='#eeeeec'>Time</th><th align='left' bgcolor='#eeeeec'>Memory</th><th align='left' bgcolor='#eeeeec'>Function</th><th align='left' bgcolor='#eeeeec'>Location</th></tr>
<tr><td bgcolor='#eeeeec' align='center'>1</td><td bgcolor='#eeeeec' align='center'>0.0183</td><td bgcolor='#eeeeec' align='right'>242488</td><td bgcolor='#eeeeec'>{main}(  )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>0</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>2</td><td bgcolor='#eeeeec' align='center'>0.0412</td><td bgcolor='#eeeeec' align='right'>252064</td><td bgcolor='#eeeeec'>require_once( <font color='#00bb00'>'/var/www/html/admin/auth_verify.php'</font> )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>2</td></tr>
</table></font>
<br />
<font size='1'><table class='xdebug-error xe-warning' dir='ltr' border='1' cellspacing='0' cellpadding='1'>
<tr><th align='left' bgcolor='#f57900' colspan="5"><span style='background-color: #cc0000; color: #fce94f; font-size: x-large;'>( ! )</span> Warning: Cannot modify header information - headers already sent by (output started at /var/www/html/admin/auth_verify.php:17) in /var/www/html/admin/auth_verify.php on line <i>34</i></th></tr>
<tr><th align='left' bgcolor='#e9b96e' colspan='5'>Call Stack</th></tr>
<tr><th align='center' bgcolor='#eeeeec'>#</th><th align='left' bgcolor='#eeeeec'>Time</th><th align='left' bgcolor='#eeeeec'>Memory</th><th align='left' bgcolor='#eeeeec'>Function</th><th align='left' bgcolor='#eeeeec'>Location</th></tr>
<tr><td bgcolor='#eeeeec' align='center'>1</td><td bgcolor='#eeeeec' align='center'>0.0183</td><td bgcolor='#eeeeec' align='right'>242488</td><td bgcolor='#eeeeec'>{main}(  )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>0</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>2</td><td bgcolor='#eeeeec' align='center'>0.0412</td><td bgcolor='#eeeeec' align='right'>252064</td><td bgcolor='#eeeeec'>require_once( <font color='#00bb00'>'/var/www/html/admin/auth_verify.php'</font> )</td><td title='/var/www/html/admin/api.php' bgcolor='#eeeeec'>.../api.php<b>:</b>2</td></tr>
<tr><td bgcolor='#eeeeec' align='center'>3</td><td bgcolor='#eeeeec' align='center'>1.1052</td><td bgcolor='#eeeeec' align='right'>1227600</td><td bgcolor='#eeeeec'><a href='http://www.php.net/function.header' target='_new'>header</a>
( ??? )</td><td title='/var/www/html/admin/auth_verify.php' bgcolor='#eeeeec'>.../auth_verify.php<b>:</b>34</td></tr>
</table></font>```

**Comment by radius314** [comment]
This bug occurs because the BMLT root server (`/local_server/server_admin/json.php?admin_action=get_permissions`) has an inconsistent response contract.  If there are multiple responses, it returns an array of objects.  If there is one response, it returns just the object.

I can work around the issue by cleaning up the response in the wrapper function.

cc: @jbraswell

**Comment by radius314** [comment]
fixed in 3.0.2
